rsciio/digitalsurf/_api.py

```diff
@@ -81,26 +90,28 @@ class DigitalSurfHandler(object):
         6: "_MERIDIANDISC",
         7: "_MULTILAYERPROFILE",
         8: "_MULTILAYERSURFACE",
-        9: "_PARALLELDISC",
+        9: "_PARALLELDISC",  # not implemented
         10: "_INTENSITYIMAGE",
         11: "_INTENSITYSURFACE",
         12: "_RGBIMAGE",
-        13: "_RGBSURFACE",
-        14: "_FORCECURVE",
-        15: "_SERIEOFFORCECURVE",
-        16: "_RGBINTENSITYSURFACE",
+        13: "_RGBSURFACE",  # Deprecated
+        14: "_FORCECURVE",  # Deprecated
+        15: "_SERIEOFFORCECURVE",  # Deprecated
+        16: "_RGBINTENSITYSURFACE",  # Surface + Image
+        17: "_CONTOURPROFILE",
+        18: "_SERIESOFRGBIMAGES",
         20: "_SPECTRUM",
         21: "_HYPCARD",
     }
 
-    def __init__(self, filename=None):
+    def __init__(self, filename: str = ""):
```

Comment:
Yup it's gone. Probably will break a few notebooks here and there because I know some people explicitly load the api to use `_MS_parse` on txt file contents. But they'll figure it out

Reply:
Okay, would it worth changing `_MS_parse` to a standalone and public function?

Reply:
It is already a static method so I think we're good to go

Reply:
Users shouldn't be using private API (`rsciio.digitalsurf._api.DigitalSurfHandler._MS_parse` with 2 `_`!).

If this function is useful, it should be available in the public API, e.g. `rsciio.digitalsurf.MS_parse` with a more explicit name - what does `MS` stand for?.

Reply:
`MS` is shorthand for MicroscopeStatus. 

I renamed and exposed to public api.

Reply:
Ok it broke the tests. 

It doesn't make sense to move it to a utils sub-module because no one would understand why its there and how to import it. I burried it back in the _api module and this way people call it using `rsciio.digitalsurf._api.parse_metadata`.

This removes one `_` call from the line (and an object call), already making the world a better place wdyt?